docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#change-or-delete-response-headers' %}
+
+### Change or Delete Response Headers
+
+The `RequestHook._onConfigureResponse()` method sets and removes headers, effectively mocking part of the response.
```

Comment:
> The `RequestHook._onConfigureResponse()` method sets and removes headers, effectively mocking part of the response.

This method do nothing. You can inside this method change headers.
```js
_onConfigureResponse (event) {
    super._onConfigureResponse(event);
    event.setHeader('x-token', 'token-123');
    event.removeHeader('x-frame-options');
  }
```

> effectively mocking part of the response.

The `mock` means `create something synthetic instead of real`. 
The HTTP response contains the statusCode, headers and body.
You can modify the part of the existing response (only headers).

Reply:
> You can set or remove response headers in a `RequestHook._onConfigureResponse()` call.